packages/web/src/common/store/user-list/top-supporters/errorSagas.ts

```diff
@@ -11,7 +11,6 @@ export function* handleFollowersError(action: handleTopSupportersError) {
   yield put(
     errorActions.handleError({
       message: action.type,
-      shouldRedirect: true,
```

Comment:
I think I missed something, but it seems like `shouldRedirect` controlled whether we show the error screen or not and I am not seeing that distinction between errors anymore?

Reply:
You were definitely correct here. Put back the "shouldRedirect" and switched all invocations of openErrorPage to use the common handleError, which will allow us to toast or redirect going forward